CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Releases
 
+## v7.0.2
+- **FIX**
+  - COrrected logic causing the backup-vault-notification alerts to throw an exception when applying values to the template.
```

Comment:
Corrected logic causing the backup-vault-notification alerts to throw an exception when applying values to the template.